src/pages/studyView/charts/barChart/BarChart.tsx

```diff
@@ -140,15 +140,20 @@ export default class BarChart extends React.Component<IBarChartProps, {}> implem
     @computed
     get bottomPadding(): number {
         const MAX_PADDING = 30;
-        const padding = this.tilted ? _.max(this.tickFormat.map((tick: string | string[]) => {
+        const padding = _.max(this.tickFormat.map((tick: string | string[]) => { 
             let content;
             if (_.isArray(tick)) {
                 content = tick.join();
             } else {
                 content = tick;
             }
-            return getTextWidth(content, VICTORY_THEME.axis.style.tickLabels.fontFamily, `${VICTORY_THEME.axis.style.tickLabels.fontSize}px`);
-        })) * Math.sin(Math.PI * TILT_ANGLE / 180) + 5 : 20;
+            const fontFamily = VICTORY_THEME.axis.style.tickLabels.fontFamily;
+            const fontSize = `${VICTORY_THEME.axis.style.tickLabels.fontSize}px`;
+            const textHeight = getTextHeight(content, fontFamily, fontSize);
+            const textWidth = getTextWidth(content, fontFamily, fontSize);
+            const textDiagonal = getTextDiagonal(textHeight, textWidth);
+            return 15 + (this.tilted ? textDiagonal * Math.sin(Math.PI * TILT_ANGLE / 180)  : textHeight);  
```

Comment:
Could you take a look which one is better when the reserved height is 10 vs 15? Take a look few studies. From the heroku instance, the 15 is bit too high. @brettvanderwerff

Reply:
@zhx828 I just had that because as it is written in`bottomPadding` of the master branch there is a `5px` bump on the calculated value of label lengths in addition to the `10px` reserve height factored in later on. I wasn't sure if it was there for a reason so that is why I added the `15px` to the `bottomPadding` return value. However I just changed the increase in `bottomPadding` return value from `15` to `10`. I looked at a few studies and I cant find issues with it being `10` so that is what my most recent commit has. Thanks!